Tests/Packet++Test/Tests/EthAndArpTests.cpp

```diff
@@ -7,8 +7,38 @@
 #include "ArpLayer.h"
 #include "PayloadLayer.h"
 #include "Packet.h"
+#include "OUILookup.h"
 #include "SystemUtils.h"
 
+PTF_TEST_CASE(OUILookup)
+{
+	pcpp::OUILookup lookupEngineJson;
+	PTF_ASSERT_GREATER_THAN(lookupEngineJson.initOUIDatabaseFromJson("../../3rdParty/OUIDataset/PCPP_OUIDataset.json"), 0);
+
+	PTF_ASSERT_EQUAL(lookupEngineJson.getVendorName(pcpp::MacAddress("aa:aa:aa:aa:aa:aa")), "Unknown");
+	// CIDR 36
+	PTF_ASSERT_EQUAL(lookupEngineJson.getVendorName(pcpp::MacAddress("70:B3:D5:2A:B0:00")), "NASA Johnson Space Center");
+	PTF_ASSERT_EQUAL(lookupEngineJson.getVendorName(pcpp::MacAddress("70:B3:D5:2A:BF:FF")), "NASA Johnson Space Center");
+	// CIDR 28
+	PTF_ASSERT_EQUAL(lookupEngineJson.getVendorName(pcpp::MacAddress("f4:0e:11:f0:00:00")), "Private");
+	PTF_ASSERT_EQUAL(lookupEngineJson.getVendorName(pcpp::MacAddress("f4:0e:11:ff:ff:ff")), "Private");
+	// Short
+	PTF_ASSERT_EQUAL(lookupEngineJson.getVendorName(pcpp::MacAddress("00:08:55:01:01:01")), "NASA-Goddard Space Flight Center");
+
+	pcpp::OUILookup lookupEngineInternal;
+	PTF_ASSERT_GREATER_THAN(lookupEngineInternal.initOUIDatabaseFromInternalData(), 0);
+
+	PTF_ASSERT_EQUAL(lookupEngineInternal.getVendorName(pcpp::MacAddress("aa:aa:aa:aa:aa:aa")), "Unknown");
+	// CIDR 36
+	PTF_ASSERT_EQUAL(lookupEngineInternal.getVendorName(pcpp::MacAddress("70:B3:D5:2A:B0:00")), "NASA Johnson Space Center");
+	PTF_ASSERT_EQUAL(lookupEngineInternal.getVendorName(pcpp::MacAddress("70:B3:D5:2A:BF:FF")), "NASA Johnson Space Center");
+	// CIDR 28
+	PTF_ASSERT_EQUAL(lookupEngineInternal.getVendorName(pcpp::MacAddress("f4:0e:11:f0:00:00")), "Private");
+	PTF_ASSERT_EQUAL(lookupEngineInternal.getVendorName(pcpp::MacAddress("f4:0e:11:ff:ff:ff")), "Private");
+	// Short
+	PTF_ASSERT_EQUAL(lookupEngineInternal.getVendorName(pcpp::MacAddress("00:08:55:01:01:01")), "NASA-Goddard Space Flight Center");
```

Comment:
nit: instead of duplicating the tests we can do something like this:
```cpp
pcpp::OUILookup* OUILookups[2];
OUILookups[0] = &lookupEngineJson;
OUILookups[1] = &lookupEngineInternal;

for (auto ouiLookup : ouiLookups)
{
	PTF_ASSERT_EQUAL(ouiLookup->getVendorName(...));
	PTF_ASSERT_EQUAL(ouiLookup->getVendorName(...));
	PTF_ASSERT_EQUAL(ouiLookup->getVendorName(...));
	...
}
```